app/controllers/util/CustomMetadataUtils.scala

```diff
@@ -40,7 +40,7 @@ class CustomMetadataUtils(allCustomMetadataProperties: List[CustomMetadata]) {
 
 //    Use this until descriptions are added, then use property.description.getOrElse("")
     val fieldLabel = dbAndFieldLabel.getOrElse(property.name, property.fullName.getOrElse(""))
-    val fieldDescription = dbAndFieldDescription.getOrElse(property.name, property.description.getOrElse(""))
+    val fieldDescription = property.description.getOrElse("")
```

Comment:
Thanh, can you please remove `dbAndFieldDescription` map if we are not using this anywhere else?

https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/app/controllers/util/CustomMetadataUtils.scala#L20

Reply:
And also the comment on line 41

Reply:
It's not being used. I have removed it